doc/devbox_setup.md

```diff
@@ -151,6 +151,9 @@ This section describes how to set up a development environment for the C SDK on
   sudo apt-get update
   sudo apt-get install -y git cmake build-essential curl libcurl4-openssl-dev libssl-dev uuid-dev
   ```
+  NOTE: If you are planning to use HTTP with wolfSSL, you must configure curl before installation.
+  1. Download the latest [curl](https://github.com/curl/curl/releases).
+  2. See curl [documentation](https://curl.se/docs/install.html) to use the wolfSSL option for configuration.  Install.
```

Comment:
few other things that can be added in this is 
1. Curl support with wolfssl
https://www.wolfssl.com/products/curl/
2. Which options are required by azure from wolfssl  lib : renegotiation

Reply:
good idea.  I'll do another PR since this is merged.